### ISSUE TITLE
Generation time increased

### ISSUE THREAD

**Issue by kri1927** [comfyanonymous/ComfyUI#937]
It happens again... (similar to this https://github.com/comfyanonymous/ComfyUI/issues/858)  but this time, instead of increasing tremendously it gets up a bit but it's not that much. I started noticing this issue about 2 days ago when I usually take about 170s-180s (30s on first gen and 1:30 on upscale) about 2 minutes or more to complete one generation but now about 2 days ago it goes up to 220s-250s (30-50s on first gen and 2:30 on upscale)
to put it simply it goes from 1 s/it > 2+ s/it and 7 s/it > 10-15 s/it
updating doesn't seem to help anymore...
(just FYI before this happen I installed a tiled Ksampler and controlnet, preprocessor but ii think this is not relevant since reinstalling only just comfyui alone on other folders and running it doesn't decrease any time used)

**Comment by AZmisc** [comment]
Same here. A workflow with upscaler time 2-5x longer than a few days ago. First I saw this in workflows with OpenPose controlnet, now these delays intruded even the regular workflows. These used to take 200 seconds all in, now looking at 800 seconds.

**Comment by comfyanonymous** [comment]
What's your GPU, amount of ram and OS? Are you using the standalone? Is it updated to the latest?

**Comment by AZmisc** [comment]
> What's your GPU, amount of ram and OS? Are you using the standalone? Is it updated to the latest?

My install of CUI is deployed via "git-install" and it auto-updates everytime I launch. So, I do have the very latest from github. Most of the time generating a single file when I launch CUI is fine. It is the subsequent generations that end up taking twice, thrice, etc. as long as the first generation took.

<details>
<summary>GPU, OS, RAM</summary>
![DX p1](https://github.com/comfyanonymous/ComfyUI/assets/47092625/886cf987-4fc1-4768-a141-69fb718c9f4f)

![DX p2](https://github.com/comfyanonymous/ComfyUI/assets/47092625/c83facbd-e3a7-4042-8c2d-395ae121fb09)
</details>

<details>
<summary>Workflow File</summary>
![a_13517_](https://github.com/comfyanonymous/ComfyUI/assets/47092625/b25be90e-af6f-4960-9ca2-0000756fee19)

This file took 11 minutes to render, tho normally it renders under 3.
</details>

**Comment by AZmisc** [comment]
As a test I removed LORAs from the workflow, and it took ~2 minutes. Strange, but there might be a bottleneck in LORAs. Here is the workflow file without LORA nodes:

<details>
<summary>Prompt</summary>

![a_13518_](https://github.com/comfyanonymous/ComfyUI/assets/47092625/41cb5556-0833-4c99-bcee-f67ac83b22a2)

</details>

**Comment by kri1927** [comment]
![CPU-Z  7_24_2023 3_15_00 PM](https://github.com/comfyanonymous/ComfyUI/assets/62532921/af752ae7-b009-4cdd-b447-beec06c26677)
![CPU-Z  7_24_2023 3_14_48 PM](https://github.com/comfyanonymous/ComfyUI/assets/62532921/205094d8-5d50-416c-8ac4-14c0ba6dfa31)
![ComfyUI_0001](https://github.com/comfyanonymous/ComfyUI/assets/62532921/626f70d0-2a33-4abc-a9bd-0698ab3e34e8)
this is the workflow, It used to be only 1:30 on upscaling but now it always be 2:30 this is usable but I want the previous speed back T_T

**Comment by kri1927** [comment]
> Try launching comfyui with: --disable-cuda-malloc

Will that decrease my generation's max resolution?

**Comment by kri1927** [comment]
OH hey I just found this workflow from some dude on cutoff node and I tried it and turned out it make the speed goes normal for me could this be a problem that comes from some node?
Mine (same as above (the horse)
![ComfyUI_0001](https://github.com/comfyanonymous/ComfyUI/assets/62532921/bbe22bb2-7d45-4987-b95c-6a998a2ba44a)
His
![ComfyUI_00080_](https://github.com/comfyanonymous/ComfyUI/assets/62532921/a5fe3cf9-3ec0-441b-bde5-21f9d35f3e5d)
and this is the speed
![miii](https://github.com/comfyanonymous/ComfyUI/assets/62532921/d92a1e58-d698-4ec8-b01d-21d54fc8a72c)
![hiss](https://github.com/comfyanonymous/ComfyUI/assets/62532921/e0913c38-dd7b-4212-824c-80be9491070b)

**Comment by kri1927** [comment]
I think I figured the cause of the problem and that is the condition combine node.
idk why but it seems like it goes away but not really, like from the horse workflow https://github.com/comfyanonymous/ComfyUI/issues/937#issuecomment-1647448759 
its speed is 2:40+ minute right? even though I delete every group and left just the generation and upscaler speed was still the same when I delete the condition combine the speed goes up and left only 1:40 just like what I needed but when I got back to my old generation that I used it since the first day and load it use about 3 minute (but this one use Dpm2s ancestral karras instead of DDIM simple both in generation and upscaler, I know it's going to take more time but to go up from 1:40+> 3:00 I don't think it was like this before so to confirm it I loaded the horse workflow and use that sampler on the one with no condition combine and it turned out to be 3 minutes also.
So is this normal that condition combine make speed goes slower like this? also dpm2s ancestral karras sampler that make it goes up almost 3 times the time consumed?
